site/docs/v1/tech/core-concepts/users.adoc

```diff
@@ -269,3 +280,19 @@ The scheduling software could know that Richard would have access to the schedul
 
 If you were to model this using only applications, you'd have to have twenty applications in FusionAuth (two for each store) and keeping those configurations synchronized might be difficult.
 And if you added more applications or stores, you'd face a combinatorial explosion of applications.
+
+=== The user.data Field
+
+You can add add arbitrary JSON to the `user.data` field, such as `org_type` and `org_id` fields. You can do this using the link:/docs/v1/tech/apis/users[User API] on user creation.
+
+This can then be read in a link:/docs/v1/tech/lambdas/jwt-populate[JWT Populate Lambda] and pushed into the tokens generated during authentication.
+
+The downstream application can examine tokens and determine to which organizations a user has access.
```

Comment:
This read a bit awkward to me. I tried a search to find alternatives, and it turns out that there is [no rule about ending a sentence with a preposition](https://www.merriam-webster.com/grammar/prepositions-ending-a-sentence-with)!

If you had another reason for the phrasing or prefer to keep it this way, that's cool too.

Reply:
Awesome, didn't know that!